### PR TITLE
Update index.md to match the README

### PR DESCRIPTION
**Pull request by alevy713**

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -1,18 +1,29 @@
----
-# generated by https://github.com/hashicorp/terraform-plugin-docs
```
Comment (Khyme) [inline docs/index.md:2]: so this one won't be generated anymore ?

**Review by Khyme — APPROVED** [review]

**Comment by Khyme** [comment]
I don't really understand, doas that mean we have to maintain both or one is generated from the other?

File: docs/index.md (hunk 1 above)
Comment (alevy713) [reply docs/index.md:2]: Yes it'll come from the template file (maybe there's a way to embed the README in the template)

**Comment by alevy713** [comment]
> I don't really understand, doas that mean we have to maintain both or one is generated from the other?

Yes both need to be maintained. IMO the generated index.md isn't super helpful for getting started. I don't expect it to change much either.

**Review by itroot — APPROVED** [review]

**Pull request merged**